config/prow/cluster/monitoring/prometheus_operator_deployment.yaml

```diff
@@ -18,12 +18,11 @@ spec:
     spec:
       containers:
       - args:
+        - --log-level=all
         - --kubelet-service=kube-system/kubelet
-        - --logtostderr=true
-        - --config-reloader-image=quay.io/coreos/configmap-reload:v0.0.1
```

Comment:
This is the only removal that stands out to me, but I figure it is intentional. Please proceed if so.

Reply:
Yes it is, and it's so obviously noted on the branch
![Screen Shot 2021-03-09 at 3 05 13 PM](https://user-images.githubusercontent.com/45011425/110550627-e81e4d00-80e8-11eb-8c54-b061e3f5af19.png)


https://github.com/prometheus-operator/prometheus-operator/blob/release-0.45/cmd/operator/main.go